capabilities.json

```diff
@@ -397,6 +397,28 @@
                 }
             }
         },
+        "taskConfig": {
```

Comment:
Please include displayNameKEy for all of new properties. Please also update resource file for en-us locale